cl/recap/mergers.py

```diff
@@ -701,7 +703,7 @@ def add_docket_entries(d, docket_entries, tags=None):
             date_last_filing=max(known_filing_dates)
         )
 
-    return rds_created, content_updated
+    return rds_created, content_updated, des_returned
```

Comment:
We need to update the docstring for this tweak.

Reply:
Yeah, updated!